GOVERNANCE.md

```diff
@@ -0,0 +1,179 @@
+the organization.
+
+The Core Team should have occasional meetings to discuss the project, with meeting minutes
+being added to [eslint-community/governance](https://github.com/eslint-community/governance).
+
+The Core Team are together with the ESLint TSC the organization owners and the only ones
+with `admin` access to repositories within the organization.
+
+The Core Team are the only members of the `@eslint-community/core-team` team.
+
+## ESLint TSC
+
+The ESLint TSC are the ultimate caretakers of the ESLint-Community, appoints the Core
+Team and holds the ownership of the `eslint-community-bot` npm account that owns all
+the ESLint-Community npm modules.
+
+The Core Team are together with the ESLint TSC the organization owners and only ones
+with `admin` access to repositories within the organization.
+
+The ESLint TSC are the only members of the `@eslint-community/eslint-tsc` team.
+
+## Collaborators
+
+Collaborators maintain the projects of the ESLint-Community organization.
+
+Collaborators are added on a project by project basis by adding them to a team of
+the same name as its main project repository, eg. `@eslint-community/eslint-plugin`.
+
+A project team is given `write` level access to its repositories.
+
+If multiple repositores are needed by the same project they should all be added to
+the project team.
+
+Both Collaborators and non-Collaborators may propose changes to the source code
+of the projects of the organization. The mechanism to propose such a change is a
+GitHub pull request. Collaborators review and merge (_land_) pull requests
+following the [CONTRIBUTING](CONTRIBUTING.md) guidelines.
+
+### Collaborator activities
+
+* Helping users and novice contributors
+* Contributing code and documentation changes that improve the project
+* Reviewing and commenting on issues and pull requests
+* Merging pull requests
+* Release plugins
+
+The Core Team can remove inactive Collaborators or provide them with
+_Past Collaborators_ status. Past Collaborators may request that the Core
+Team restore them to active status.
+
+## Projects
+
+There are two kinds of projects in ESLint Community: Primary and auxiliary projects.
+
+Primary projects are maintained by Core Team in collaboration with project Collaborators
+and will be ensured to be kept up to date at a best effort pace by the Core Team.
+
+Auxiliary projects are maintained by its project Collaborators and is kept up to date at
+whatever pace those Collaborators chose. The Core Team merely fascilitates nominations
+of new Collaborators to such projects, ensuring there's a path to keep it maintained,
+without making any promises.
+
+All project npm modules should be owned (and solely owned) by the `eslint-community-bot`
+user on npm and automatic releases should be set up by the Core Team using the organization
+secret in the GitHub organization.
+
+Additional assets such as domain names, web sites etc should also be in control of the
+Core Team and/or the ESLint TSC.
```

Comment:
Should probably add that forks should preferably be published as @eslint-community/<original-name>

Reply:
Added it